package.json

```diff
@@ -18,6 +18,7 @@
 		"@playwright/test": "^1.28.1",
 		"@sveltejs/adapter-auto": "^2.0.0",
 		"@sveltejs/kit": "^1.27.4",
+		"@types/geojson": "^7946.0.14",
```

Comment:
wow first time i see a version be so high 😮

Reply:
I know, particularly bizarre for a package that is just the types of a standardised data format. Suspicious...